WebDriverAgentTests/IntegrationTests/XCUIElementFBFindTests.m

```diff
@@ -61,6 +61,15 @@ - (void)testDescendantsWithXPathQuery
   XCTAssertEqualObjects(matchingSnapshots.lastObject.label, @"Alerts");
 }
 
+- (void)testDescendantsWithMoreComplexXPathQuery
```

Comment:
Will do. I didn't seem to find a straightforward way to check for visibility. Am I missing something here?